### ISSUE TITLE
 chr7:g.43798425C>T 

### ISSUE THREAD

**Issue by pnrobinson** [charite/jannovar#200]
This variant gets reported as 5' UTR.
Actually, it is located between exons 1 and 2 (both of which code for the 5' UTR) as 
uc003tir.3:c.-22+92C>T:p.(=) 
Question: Jannovar or Exomiser bug?

**Comment by visze** [comment]
Jannovar BUG:

```
java -jar jannovar-cli-0.16.jar annotate-pos -d data/hg19_ucsc.ser -c "chr7:43798425C>T"
```
leads to
```
chr7:43798425C>T	FIVE_PRIME_UTR_VARIANT,FIVE_PRIME_UTR_VARIANT	BLVRA:uc003tir.3:c.-22+92C>T:p.(=),BLVRA:uc010kxv.3:c.-116+92C>T:p.(=)
```

**Comment by holtgrewe** [comment]
This appears to be a bug in Jannovar. The code [here](https://github.com/charite/jannovar/blob/master/jannovar-core/src/main/java/de/charite/compbio/jannovar/annotation/builders/AnnotationBuilder.java#L231) does not distinguish between exons and introns.

**Comment by holtgrewe** [comment]
Should be fixed? At least it is fixed after #210 (probably also earlier).
